<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editAction</class>
 <widget class="QDialog" name="editAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>512</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>édition d'action</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="grb_Pos">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Position :</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="qhl_Altitude">
        <item>
         <widget class="QLabel" name="lbl_Altitude">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Altitude  :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_Altitude">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_Longitude">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Liberation Mono</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Longitude :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_Longitude">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkOnGround">
        <property name="text">
         <string>Au sol</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lbl_Latitude">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Liberation Mono</family>
           </font>
          </property>
          <property name="text">
           <string>Latitude  :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_Latitude">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chk_Smoke">
        <property name="text">
         <string>Fumée</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="qhs_Smoke">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_Smoke_1">
            <property name="text">
             <string>Peu de fumée</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lbl_Smoke_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fumée abondante</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="grpType">
       <property name="title">
        <string>Type d'action :</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rdbApparition">
          <property name="text">
           <string>Apparition</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rdbDisparition">
          <property name="text">
           <string>Disparition</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rdbDeplacement">
          <property name="text">
           <string>Déplacement</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="rdbChangementEtat">
          <property name="text">
           <string>Changement d'état</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Objet :</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lbl_Name">
            <property name="text">
             <string>Nom :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="qle_Name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbl_Id">
            <property name="text">
             <string>Identifiant :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="qle_Id">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rotation :</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lbl_Psi">
          <property name="text">
           <string>Psi   :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_Psi">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lbl_Theta">
          <property name="text">
           <string>Theta :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_Theta">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="lbl_Phi">
          <property name="text">
           <string>Phi   :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qle_Phi">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkOnGround</sender>
   <signal>toggled(bool)</signal>
   <receiver>qle_Altitude</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnGround</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Altitude</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>editAction</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Smoke</sender>
   <signal>toggled(bool)</signal>
   <receiver>qhs_Smoke</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Smoke</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Smoke_1</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_Smoke</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_Smoke_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
